django/utils/http.py

```diff
@@ -315,8 +319,15 @@ def _is_safe_url(url, host):
     # URL and might consider the URL as scheme relative.
     if unicodedata.category(url[0])[0] == 'C':
         return False
+    scheme = url_info.scheme
+    # If we have a URL like //example.com/p - it doesn't have a scheme but does
```

Comment:
more simply: `Consider URLs without a scheme (e.g. //example.com/p) to be http.`